op-e2e/e2eutils/challenger/helper.go

```diff
@@ -96,32 +77,33 @@ func applyCannonConfig(
 	genesisBytes, err := json.Marshal(l2Genesis)
 	require.NoError(err, "marshall l2 genesis config")
 	genesisFile := filepath.Join(c.Datadir, "l2-genesis.json")
-	require.NoError(os.WriteFile(genesisFile, genesisBytes, 0644))
+	require.NoError(os.WriteFile(genesisFile, genesisBytes, 0o644))
 	c.CannonL2GenesisPath = genesisFile
 
 	rollupBytes, err := json.Marshal(rollupCfg)
 	require.NoError(err, "marshall rollup config")
 	rollupFile := filepath.Join(c.Datadir, "rollup.json")
-	require.NoError(os.WriteFile(rollupFile, rollupBytes, 0644))
+	require.NoError(os.WriteFile(rollupFile, rollupBytes, 0o644))
 	c.CannonRollupConfigPath = rollupFile
 }
 
-func WithOutputCannon(
+func WithCannon(
 	t *testing.T,
 	rollupCfg *rollup.Config,
 	l2Genesis *core.Genesis,
 	rollupEndpoint string,
-	l2Endpoint string) Option {
+	l2Endpoint string,
+) Option {
 	return func(c *config.Config) {
-		c.TraceTypes = append(c.TraceTypes, config.TraceTypeOutputCannon)
+		c.TraceTypes = append(c.TraceTypes, config.TraceTypeCannon)
 		c.RollupRpc = rollupEndpoint
 		applyCannonConfig(c, t, rollupCfg, l2Genesis, l2Endpoint)
 	}
 }
 
-func WithOutputAlphabet(alphabet string, rollupEndpoint string) Option {
+func WithAlphabet(alphabet string, rollupEndpoint string) Option {
 	return func(c *config.Config) {
-		c.TraceTypes = append(c.TraceTypes, config.TraceTypeOutputAlphabet)
+		c.TraceTypes = append(c.TraceTypes, config.TraceTypeAlphabet)
 		c.RollupRpc = rollupEndpoint
 		c.AlphabetTrace = alphabet
```

Comment:
We shouldn't have ever been setting the `AlphabetTrace` option for output_alphabet type - I think we forgot to update this when the alphabet VM started using the l2 block number as the starting input.  We can just remove the `alphabet` param from this method and places that call it when we remove the `AlphabetTrace` config option and flag.